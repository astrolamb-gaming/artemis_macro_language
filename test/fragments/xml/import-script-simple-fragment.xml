<mission_data version="1.0">
    <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
    <imports>
        <import name="script-simple-fragment.xml" />
    </imports>
    <start>
        <expand _template="test"/>
       
    </start>
</mission_data> 