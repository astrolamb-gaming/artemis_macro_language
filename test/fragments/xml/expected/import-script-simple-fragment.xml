<mission_data version="1.0">
  <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
  <start>
    <big-message/>
    <big-message title="Hello, Javascript"/>
    <big-message title="inline Hello, Javascript"/>
    <big-message title="Instance Hello, Plugin"/>
    <big-message title="Property Hello, property"/>
    <big-message title="function Hello, Function"/>
  </start>
</mission_data>